commands/diceRoller.js

```diff
@@ -124,13 +124,14 @@ function parseDiceCommand(command) {
   let dice = new Object();
   const commandCleaned = command.toLowerCase().split(' ').join('').split('r').join(''); // removes spaces and Rs
   let [amount, diceConfig] = commandCleaned.split('d');
-  let [sides, modifier] = diceConfig.split('+') || diceConfig.split('-');
+  let [sides, modifier] = diceConfig.includes('+') ? diceConfig.split('+') : diceConfig.split('-');
```

Comment:
I know you don't like regex, but you can do this way easier.

```suggestion
  let [sides, modifier] = diceConfig.split(/[+-]/);
```

Reply:
Ok I added the regex, and I'm going to try and stop being a curmudgeon about it. :D but only in spots that I can follow what its doing :P